docs/sources/mobile-app/push-notifications/index.md

```diff
@@ -83,3 +84,18 @@ You can also enable or disable Do Not Disturb override for **important** notific
 On-call shift notifications are sent to announce upcoming shifts, roughly ~15 minutes in advance.
 
 To enable or disable on-call shift notifications, use the **On-call shift notifications** section in the **Push notifications** settings.
+
+### Shift swap notifications
+
+Shift swap notifications are generated when a [shift swap request][shift-swaps] is created,
+informing all users in the on-call schedule (except the initiator) about potential schedule changes.
```

Comment:
the notification serves much more as an opportunity to help out your team member, rather then becoming aware of potential (future) changes